l10n_ch_dta/wizard/create_dta.py

```diff
@@ -671,15 +672,15 @@ def _process_payment_lines(self, data, pline, elec_context, seq):
             elec_context['partner_bvr'] = part
         self._set_bank_data(pline, elec_context, seq)
 
-        dt = fields.Datetime
-        date_to_datetime = lambda date: dt.to_string(dt.from_string(date))
         if pline.order_id.date_scheduled:
-            date_value = date_to_datetime(pline.order_id.date_scheduled)
+            date_value = datetime.strptime(pline.order_id.date_scheduled,
+                                           '%Y-%m-%d')
         elif pline.date:
-            date_value = date_to_datetime(pline.date)
+            date_value = datetime.strptime(pline.date,
+                                           '%Y-%m-%d')
```

Comment:
You should use `fields.Date.from_string(pline.date)` instead of hardcoding the format.